apps/client/src/i18n/en.json

```diff
@@ -74,17 +74,20 @@
   "CANVAS:DELETE": "Delete",
   "CANVAS:FLOW-PARAMS-INPUT-OUTPUT": "Input Output",
   "CANVAS:FLOW-NAME-EXISTS":"Flow with name \"{{value}}\" already exists, please choose another name.",
+  "CANVAS:STREAM-NAME-EXISTS":"Stream with name \"{{value}}\" already exists, please choose another name.",
```

Comment:
I think the naming structure we currently follow for the strings is confusing, the whole file look disorganized. Adding stream specific streams in between flow strings is only going to make it harder to find where a string is supposed to be used.

I think we should follow a structure like:
`<package>:<submodule>:<string specific>`

for example `STREAMS:DESIGNER:STREAM-NAME-EXISTS`

or we can make use of the library namespacing as well, something like:

```jsonc
{
 "STREAMS": {
    "DESIGNER:ERROR-STREAM-NAME-EXISTS"
  }
}

```

and used for example in a template:
```html
<div>{{ 'STREAMS.DESIGNER:ERROR-STREAM-NAME-EXISTS' | translate }}</div>
```